cf-macro/index.js

```diff
@@ -0,0 +1,83 @@
+'use strict'
+
+const _ = require('lodash')
+const Ajv = require('ajv')
+
+const alarms = require('slic-watch-core/alarms')
+const dashboard = require('slic-watch-core/dashboard')
+const CloudFormationTemplate = require('slic-watch-core/cf-template')
+const defaultConfig = require('slic-watch-core/default-config')
+const { slicWatchSchema } = require('slic-watch-core/config-schema')
+
+exports.handler = async function (event, context) {
+  const response = processFragment(event)
+  return response
+}
```

Comment:
Is it worth doing a function if we are only calling another one and returning its result?

Could we just move the entire code of `processFragment` inside here?

Reply:
Sure, moved the code.